backend/migrations/20201009195058_statuses.js

```diff
@@ -0,0 +1,14 @@
+const { tableStatuses } = require('../src/models/status')
+
+exports.up = (knex) => {
+  //knex.schema.dropTableIfExists('statusupdates', 'students', 'statuses')
+  return knex.schema.createTable(tableStatuses, (table) => {
+    table.increments('StatusID').primary() // ToDo StatusID vs ID (postgres issue). Figure out uuid
```

Comment:
Remove this TODO comment.
```suggestion
    table.increments('StatusID').primary()
```